addon/loaders/utilities.js

```diff
@@ -37,3 +37,10 @@ export function nodeLoader(loader) {
     return () => RSVP.resolve();
   }
 }
+
+export function uriIncluded (uri, document) {
+  if (document.scripts) {
+    return Array.from(document.scripts).filter(script => script.src && script.src.includes(uri));
```

Comment:
I _thought_ that `document.scripts` was supported waaaay back in IE6ish timeframe, so I would have assumed we could not rely on `Array.from`